input/ig.xml

```diff
@@ -1426,79 +1278,6 @@
 			<description value="RelatedPerson: Mr. Peter Rabbit&#xa;&#xa;Relationship to Patient: Emergency Contact and Unrelated Friend of Mr Li Wang"/>
 			<exampleCanonical value="http://hl7.org/fhir/StructureDefinition/RelatedPerson"/>
 		</resource>
-		<!--SERVICEREQUEST EXAMPLES-->
```

Comment:
There are still RelatedPerson examples - where are these used in other examples?

Reply:
They were referenced by vital signs profiles (height & weight). Removed RelatedPerson examples and updated examples they were referenced by (subject to further change in the example cleanup)

Reply:
No need to remove them and change the examples they were still representative of AU Core. Will resolve this PR anyway so we don't spend more time. Majority of examples will change in a forthcoming PR.